tests/model_serving/maas_billing/body_base_routing/pre_auth_model_header/test_bbr_pre_auth_inference.py

```
@@ -63,6 +63,66 @@ def test_inference_model_in_body_only_invalid_key_returns_401(
+            url=bbr_inference_url,
+            headers=bbr_api_key_headers,
+            json=streaming_payload,
+            timeout=60,
+            stream=True,
+        )
+        assert response.status_code == 200, f"Expected 200 for streaming BBR inference, got {response.status_code}"
+        content_type = response.headers.get("content-type", "")
+        assert "text/event-stream" in content_type, (
+            f"Expected text/event-stream content type for streaming response, got '{content_type}'"
+        )
+        chunks = [line for line, _ in zip(response.iter_lines(), range(10)) if line]
```

Ruff (0.15.18)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)